saos-enrichment/src/main/java/pl/edu/icm/saos/enrichment/hash/MarkNotProcessedAsNotIndexedReader.java

```diff
@@ -0,0 +1,64 @@
+package pl.edu.icm.saos.enrichment.hash;
+
+import java.util.LinkedList;
+
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemStreamException;
+import org.springframework.batch.item.ItemStreamReader;
+import org.springframework.batch.item.NonTransientResourceException;
+import org.springframework.batch.item.ParseException;
+import org.springframework.batch.item.UnexpectedInputException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.google.common.collect.Lists;
+
+import pl.edu.icm.saos.persistence.enrichment.JudgmentEnrichmentHashRepository;
+
+/**
+ * @author madryk
+ */
+@Service
+public class MarkNotProcessedAsNotIndexedReader implements ItemStreamReader<Long> {
```

Comment:
The name is a little misleading. It's part of processing after the tag upload and says about not processed. Not processed what? What about MarkChangedTagJudgmentsAsNotIndexedReader?